hydra-core/spec/lib/hydra_generator_notspec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
```

Comment:
Can you say something about why this is a "`notspec`"? are we just trying to ignore it?